src/Files.App/UserControls/TabBar/TabBar.xaml

```diff
@@ -58,13 +58,70 @@
 		</ResourceDictionary>
 	</local:BaseTabBar.Resources>
 
-	<Border
+	<!--  TODO: Use TitleBar of WinAppSdk 1.6 when available  -->
+	<Grid
 		x:Name="TitlebarArea"
 		HorizontalAlignment="Stretch"
 		x:FieldModifier="public"
 		Background="Transparent">
+		<Grid.ColumnDefinitions>
+			<ColumnDefinition Width="Auto" />
+			<ColumnDefinition Width="Auto" />
+		</Grid.ColumnDefinitions>
+
+		<!--  Header Button  -->
+		<Button
+			Width="32"
+			Height="32"
+			Margin="4,4,0,-4"
+			Padding="0"
+			Background="Transparent"
+			BorderThickness="0">
+			<FontIcon FontSize="14" Glyph="&#xEA3A;" />
+
+			<Button.Flyout>
+				<MenuFlyout Placement="BottomEdgeAlignedLeft">
+					<MenuFlyoutItem Text="New window">
+						<MenuFlyoutItem.Icon>
+							<FontIcon Glyph="&#xEA3A;" />
```

Comment:
This PR will probably be ready before the updated icons, if it helps, I can add the icons using the current system so we can move forward with this without delay.

Reply:
Good idea.
I have to create a modified style of MenuFlyoutItem for custom icons as we did for images.

Reply:
We can do that, or we can create a custom flyout because there is a good chance that we'll add more UI elements in the future.

Reply:
I'm not sure what you meant by custom fly out.
And what kind of UI elements? I can think of it.
Restyling should resolve such issues mostly.

Reply:
I mean regular `Flyout` instead of `MenuFlyout`.

> And what kind of UI elements? I can think of it.

#11900

Reply:
Not sure I can use Flyout and buttons on it.

https://github.com/files-community/Files/issues/10291
As well as MenuFlyoutItem, we have the same issue in CommandBarButton. So restyling is the best way for now.

Reply:
Not sure I can use Flyout and buttons on it. Templating and restyling should solve that problem though.

https://github.com/files-community/Files/issues/10291
As well as MenuFlyoutItem, we have the same issue in CommandBarButton. So restyling is the best way for now.

Reply:
I have a basic idea on how to do the styling, let's focus on the code behind and revisit the styling once everything else is ready.